src/prettier.config.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line no-undef
+module.exports = {
```

Comment:
Why did you add multiple configurations for `prettier` ??

Reply:
removed